npm/private/npm_translate_lock.bzl

```diff
@@ -868,14 +906,27 @@ load("@aspect_rules_js//npm/private:npm_package_store.bzl", _npm_package_store =
 
         _registry = url.split("//", 1)[-1]
         npm_token = None
+        npm_auth_username = None
+        npm_auth_password = None
         match_len = 0
         for (auth_registry, auth_token) in npm_tokens.items():
             if _registry.startswith(auth_registry) and len(auth_registry) > match_len:
                 npm_token = auth_token
                 match_len = len(auth_registry)
 
+        if not npm_token:
```

Comment:
I'd drop the `if not npm_token:` here. Let npm_import deal with the case of both being set.

It looks like right now it will give preference to `npm_auth`. Should it fail or warn if both happen to be set in the `.npmrc` for a package or just silently go with one of them>